sdk/storage/azure-storage-blob/azure/storage/blob/_blob_client.py

```diff
@@ -736,6 +740,10 @@ def get_blob_properties(self, **kwargs):
             Required if the blob has an active lease. Value can be a BlobLeaseClient object
             or the lease ID as a string.
         :paramtype lease: ~azure.storage.blob.BlobLeaseClient or str
+        :keyword str version_id:
+            The version id parameter is an opaque DateTime
+            value that, when present, specifies the version of the blob to delete.
+            It for service version 2019-10-10 and newer.
```

Comment:
"Introduced in service version 2019-10-10"
Also add the docstring tag for new parameters in the SDK.